Fronius/Localization/Resources.resx

```diff
@@ -805,7 +805,7 @@
 	  <value>Default</value>
   </data>
   <data name="CableLockBehavior_AutoUnlock" xml:space="preserve">
-	  <value>Always unlock, wenn not charging</value>
+	  <value>Always unlock</value>
```

Comment:
```suggestion
	  <value>Always unlock if not charging</value>
```
"if not charging" has to remain because the cable is always locked while charging is in progress